soc/nxp/kinetis/ke1xz/Kconfig

```diff
@@ -11,5 +11,4 @@ config SOC_SERIES_KE1XZ
 	select CPU_CORTEX_M_HAS_VTOR
 	select CLOCK_CONTROL
 	select HAS_MCUX
-	select PLATFORM_SPECIFIC_INIT
-	select HAS_PM
```

Comment:
I think this is a cut and paste error - why is HAS_PM being removed for this soc only?

Reply:
yeah, rebase issue, fixed.